extensions/sql-database-projects/src/dialogs/utils.ts

```diff
@@ -38,58 +40,106 @@ export function getPublishServerName(target: string): string {
 	return target === constants.targetPlatformToVersion.get(SqlTargetPlatform.sqlAzure) ? constants.AzureSqlServerName : constants.SqlServerName;
 }
 
+/**
+ * Returns the list of image tags for given target
+ * @param imageInfo docker image info
+ * @param target project target version
+ * @returns image tags
+ */
+export async function getImageTags(imageInfo: DockerImageInfo, target: string): Promise<string[]> {
+	let imageTags: string[] | undefined = [];
+	const versionToImageTags: Map<number, string[]> = new Map<number, string[]>();
+
+	try {
+		const imageTagsFromUrl = await HttpClient.getRequest(imageInfo?.tagsUrl, true);
+		if (imageTagsFromUrl?.tags) {
+
+			// Create a map for version and tags and find the max version in the list
+			let defaultVersion: number = 0;
+			let maxVersionNumber: number = defaultVersion;
+			(imageTagsFromUrl.tags as string[]).forEach(imageTag => {
+				const version = utils.findSqlVersionInImageName(imageTag) || defaultVersion;
+				let tags = versionToImageTags.has(version) ? versionToImageTags.get(version) : [];
+				tags = tags ?? [];
+				tags = tags?.concat(imageTag);
+				versionToImageTags.set(version, tags);
+				maxVersionNumber = version && version > maxVersionNumber ? version : maxVersionNumber;
+			});
+
+			// Find the version maps to the target framework and default to max version in the tags
+			const targetVersion = utils.findSqlVersionInTargetPlatform(constants.getTargetPlatformFromVersion(target)) || maxVersionNumber;
+
+			// Get the image tags with no version of the one that matches project platform
+			versionToImageTags.forEach((tags: string[], version: number) => {
+				if (version === defaultVersion || version >= targetVersion) {
+					imageTags = imageTags?.concat(tags);
+				}
+			});
+
+			imageTags = imageTags ?? [];
+			imageTags = imageTags.sort();
+		}
+	} catch (err) {
+		// Ignore the error. If http request fails, we just use the default tag
+		console.debug(`Failed to get docker image tags ${err}`);
+	}
+
+	imageTags.unshift(constants.dockerImageDefaultTag);
```

Comment:
it would be nice to have "Default" next to the actual tag that's the default so it's clear what default corresponds to, similar to the default for selecting target platform 
![image](https://user-images.githubusercontent.com/31145923/167042520-6e4c9cd2-bb68-42e0-b2f5-586bc9ddcc03.png)

Reply:
default is the empty string for tags that's what I added the text "default" to the list